packages/extract-react-types/src/index.js

```diff
@@ -1609,15 +1600,11 @@ function findExportedComponents(programPath, componentsToFind /*: 'all' | 'defau
   return components;
 }
 
-module.exports.findExportedComponents = (
-  programPath /*: any */,
-  typeSystem /*: 'flow' | 'typescript' */,
-  filename /*:? string */,
-  resolveOptions /*:? Object */
-) => {
-  return findExportedComponents(
-    programPath,
-    'all',
-    getContext(typeSystem, filename, resolveOptions)
-  );
-};
+export const findExportedComponents = (
+  programPath: any,
+  typeSystem: 'flow' | 'typescript',
+  filename?: string,
+  resolveOptions?: Object
+) => exportedComponents(programPath, 'all', getContext(typeSystem, filename, resolveOptions));
+
+export default extractReactTypes;
```

Comment:
My thought on this would be, make it a named export and called it something like or `extractDefaultComponentTypes` or something and maybe rename `findExportedComponents` to `extractExportedComponentTypes` since it solves the cjs usage of require().default or require() problem and it's clearer about what the exports do.

Reply:
Yeah I agree just making it a named export is better. I made the named export `extractReactTypes`.